### ISSUE TITLE
Adoption rule?

### ISSUE THREAD

**Issue by andrixnet** [opencaching/opencaching-pl#999]
Should the following caches be adoptable?
- [ ] archived
- [ ] blocked
- [ ] not yet published

**Comment by kojoty** [comment]
Hi, in my opinion: YES, why not?
 archived - someone wants to adopt and reactivate cache...
 blocked - cache is blocked because of something, but someone else want's to fix the issues and ask for unblock... 
 not yet published - someone created a cache, but because of something want to give someone else a cache - we block adopting by users under verification, so I can't see any issue...

Can you see any problem see?

**Comment by andrixnet** [comment]
Archived : either unarchive the cache, then adopt, or adopt as archived, the end result is the same, so there is no reason to not allow it. 

Blocked : this is more delicate. @kojoty has detailed the most optimistic point of view. 
Since blocking is an administrative decision against a certain cache _OR_ a certain user, I would not allow adoption of such caches without admin intervention, considering the reason for the block to no longer be valid either prior to the request, or being invalidated by allowing such request. 
Either way, I still believe it to be an administrative decision that the reason for blocking it is no longer valid before allowing adoption.

Not yet published: allow it. 

For all adotions, I believe the following should still be observed: 
- destination user account has been properly validated (prevent adoption by users registered but not validated via the email verification system)
- destination user is allowed to hide caches (either by fulfilling the minimum required finds or being exempt from this by an admin)
- destination user is not blocked
- source user is not blocked
- if new caches (either first X, or all) require approval by OC Team, this point is more delicate. I'll explain why: 
* if cache is published and active, it already got approved by OC team, so no reason why to require approval again.
* if cache is blocked, admin (oc team) intervetion is required before allowing adoption. Again no reason to require another approval.
* if cache is archived, OC team should approve the cache (unarchival or publishing)
* if cache is not yet published, OC team is required to approve the cache according to normal new cache approval rules.
(if source user had still first X new caches requiring approval, but destination is already past that limit, still require approval or not?)
(if source user doesnt' require approval for new caches, but destination user is still within the limit of first X new caches, require approval).

What do you thing of this detailing?

**Comment by andrixnet** [comment]
IMO the adopt page should show a visual indicator of the cache status (active, inactive, archived, not yet published, blocked, etc).

**Comment by harrieklomp** [comment]
When can a user offer caches for adoption and when can a user accept a cache for adoption?
It is not clear (for me) when user can handle adoptions

- Does user need to find first X caches?
- Does user need to publish X caches?

**Comment by andrixnet** [comment]
Indeed, since there are [configurable] rules for hiding caches such as: 
- must have X finds
- new hides must be approved
- user allowed to publish without review
- user not allowed to publish

I think there should be a clear chart saying in each case how an adotion is handled.
This for the use both of the developers to write and maintain the code and the users as documentation.
